Userland/Libraries/LibMain/Main.cpp

```diff
@@ -20,6 +20,7 @@ int main(int argc, char** argv)
     auto result = serenity_main({
         .argc = argc,
         .argv = argv,
+	.envp = envp,
```

Comment:
This should be indented using spaces instead of tabs, please.

Reply:
Derp, seems vi uses tabs by default

Reply:
Easily done! I noticed this on your other new PR too.